.circleci/config.yml

```diff
@@ -21,43 +24,61 @@ workflows:
               only: /^v.+\..+\..+/
   tests:
     jobs:
-      - test_installation
+      - test-installation
 
 
 jobs:
   lint:
     docker:
-      - image: circleci/python:3.6
+      - image: cimg/python:3.6
     steps:
       - checkout
-      - run: "curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
-          | python3 - --version << pipeline.parameters.poetry_version >>"
+      - install-poetry
       - run: poetry install --no-root
       - run: poetry run isort --check .
       - run: poetry run black --check .
       - run: poetry run mypy -p "brainframe.cli"
-  upload_to_pypi:
+
+  build-deb:
+    docker:
+      - image: cimg/base:stable-18.04
+    steps:
+      - run: sudo apt-get update
+      - run: sudo apt-get install -y software-properties-common dpkg-dev devscripts equivs
+      - run: sudo add-apt-repository ppa:jyrki-pulliainen/dh-virtualenv
+      - run: sudo apt-get install -y dh-virtualenv
+      - install-poetry
+      - checkout
+      - run: cp -r package/debian .
+      - run: cp package/setup.py .
+      - run: cp package/system_package_defaults.yaml brainframe/cli/defaults.yaml
+      - run: sudo mk-build-deps --install debian/control
+      - run: poetry export --output requirements.txt
+      - run: dpkg-buildpackage --unsigned-source --unsigned-changes --build=binary
+      - run: mv ../brainframe-cli*.deb brainframe-cli.deb
+      - store_artifacts:
+          path: brainframe-cli.deb
+          destination: brainframe-cli.deb
+
+  upload-to-pypi:
     docker:
-      - image: circleci/python:3.6
+      - image: cimg/python:3.6
     environment:
       POETRY_HTTP_BASIC_PYPI_USERNAME: __token__
-
     steps:
       - checkout
-      - run: "curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
-          | python3 - --version << pipeline.parameters.poetry_version >>"
+      - install-poetry
       - run: poetry build
       - run: POETRY_HTTP_BASIC_PYPI_PASSWORD=${PYPI_PASSWORD} poetry publish
-  test_installation:
+
+  test-installation:
     machine:
       image: ubuntu-2004:202010-01
```

Comment:
In order to test the CLI, we need to run it in a VM instead of a Docker container because we need to be able to install Docker in the system we're testing. We're not using a VM to build, though, because they're slower and more expensive.